wdl/ClusterBatch.wdl

```diff
@@ -170,10 +170,15 @@ workflow ClusterBatch {
 
   output {
     File depth_vcf = ClusterDepth.clustered_vcf
```

Comment:
Missed this at first - this is one of the workflows that has a separate output JSON in Terra because the output names are the same as some of the other workflows, and that doesn't play nicely with the data tables. So when the outputs are changed like this, we either need to update the outputs JSON to add the new outputs, or (preferred) change the workflow output names to be unique - ie. merged_depth_vcf or clustered_depth_vcf or similar

Reply:
(The other workflows with Terra output JSONs did not have their outputs change in this PR)

Reply:
Actually this might be outdated because I can't find another workflow that has these output names. Regardless, because it was set up this way, either the Terra JSONs need to be updated or the WDL does

Reply:
Changed the WDL output names